### PR TITLE
[StyleCop] Fix issues in IBotkitMessageTemplate.cs

### PR DESCRIPTION
**Pull request by Konigstiger**
- Remove blank lines
- Order using directives alphabetically by namespace
- Add final period to Documentation text

Note: ‘Using directive should appear within a namespace declaration’, 'File may only contain a single type' and ‘The file header XML is invalid’ rule was ignored.

### CODE REVIEW

File: Botkit/Microsoft.BotKit/Conversation/IBotkitMessageTemplate.cs [hunk 1]
```diff
@@ -1,31 +1,78 @@
-// Copyright(c) Microsoft Corporation.All rights reserved.
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
 // Licensed under the MIT License.
 
 namespace Microsoft.BotKit.Conversation
 {
     /// <summary>
-    /// Template for defining a BotkitConversation template
+    /// Template for defining a BotkitConversation template.
```
Comment (ceciliaavila) [inline Botkit/Microsoft.BotKit/Conversation/IBotkitMessageTemplate.cs]: I think the last word is redundant. Please, remove it.

File: Botkit/Microsoft.BotKit/Conversation/IBotkitMessageTemplate.cs [hunk 2]
```diff
@@ -1,31 +1,78 @@
...
+        /// <summary>
+        /// Gets or Sets the Collect object.
+        /// </summary>
         Collect Collect { get; set; }
     }
 
+    /// <summary>
+    /// Execute Class.
+    /// </summary>
     public class Execute
```
Comment (ceciliaavila) [inline Botkit/Microsoft.BotKit/Conversation/IBotkitMessageTemplate.cs]: Please, extract this class to its own file.

File: Botkit/Microsoft.BotKit/Conversation/IBotkitMessageTemplate.cs [hunk 3]
```diff
@@ -1,31 +1,78 @@
...
+        /// <summary>
+        /// Gets or Sets the Thread.
+        /// </summary>
         public string Thread { get; set; }
     }
 
+    /// <summary>
+    /// Collect class.
+    /// </summary>
     public class Collect
```
Comment (ceciliaavila) [inline Botkit/Microsoft.BotKit/Conversation/IBotkitMessageTemplate.cs]: The same with this class.

**Comment by Konigstiger** [comment]
Changes reviewed implemented.

**Pull request merged**